x/common/denoms/denoms.go

```diff
@@ -5,6 +5,7 @@ const ( // stablecoins
 	NUSD = "unusd"
 	USD  = "uusd"
 	USDT = "uusdt"
+	USDA = "uusda"
```

Comment:
Have this written without a "u" prefix to break the curse in the price feeder. Aiming for "usda:usd" 
```suggestion
	USDA = "usda"
```